clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -691,7 +691,7 @@ ensure it is reachable, e.g. ``using N::g;``.
 Support for Reduced BMIs is still experimental, but it may become the default
 in the future. The expected roadmap for Reduced BMIs as of Clang 19.x is:
```

Comment:
```suggestion
in the future. The expected roadmap for Reduced BMIs as of Clang 20.x is:
```

Reply:
I looked up the dictionary and it says "as of" means "from". So I feel the wording is good since it was added in 19.

Reply:
Done as suggested.